Documentation/cli/README.md

```diff
@@ -340,19 +340,63 @@ Aliases: gr
 ## goroutines
 List program goroutines.
 
-	goroutines [-u (default: user location)|-r (runtime location)|-g (go statement location)|-s (start location)] [-t (stack trace)] [-l (labels)]
+	goroutines [-u|-r|-g|-s] [-t [depth]] [-l] [-with loc expr] [-without loc expr] [-group argument]
 
 Print out info for every goroutine. The flag controls what information is shown along with each goroutine:
 
-	-u	displays location of topmost stackframe in user code
+	-u	displays location of topmost stackframe in user code (default)
```

Comment:
How exactly is user vs non-user code defined?

Reply:
Not a private runtime function.

Reply:
So then any standard library function would still be user stack frame, correct?

Reply:
Yes.

Reply:
Looks like I am not the only one wondering about the distinction: https://github.com/go-delve/delve/issues/2542#issuecomment-863005481. I think the clarification you provided with a mention of scripting alternatives might be useful in the doc as well.

Reply:
Done.